util/chplenv/chpl_cpu.py

```diff
@@ -16,409 +16,19 @@
 # ARM architectures are accessed with the -mcpu= flag, except that some
```

Comment:
Should this comment also be removed?

Reply:
I'll look at moving it to a better place (we do toggle between -march and -mcpu)

Reply:
The existing comment `x86 uses -march= where the others use -mcpu=` covers it, so I'll just remove it as suggested.